public/browser-app.js

```diff
@@ -0,0 +1,87 @@
+const tasksDOM = document.querySelector('.tasks')
+const loadingDOM = document.querySelector('.loading-text')
+const formDOM = document.querySelector('.task-form')
+const taskInputDOM = document.querySelector('.task-input')
+const formAlertDOM = document.querySelector('.form-alert')
+// Load tasks from /api/tasks
+const showTasks = async () => {
+  loadingDOM.style.visibility = 'visible'
+  try {
+    const {
+      data: { tasks },
+    } = await axios.get('/api/v1/tasks')
+    if (tasks.length < 1) {
+      tasksDOM.innerHTML = '<h5 class="empty-list">No tasks in your list</h5>'
+      loadingDOM.style.visibility = 'hidden'
+      return
+    }
+    const allTasks = tasks
+      .map((task) => {
+        const { completed, _id: taskID, name } = task
+        return `<div class="single-task ${completed && 'task-completed'}">
```

Comment:
In a real frontend framework we don't do the injection like that, but it's great first try of usaging some frontend code which call the server API with axios. Have you understood this concept? that's worked?